editor/src/components/navigator/navigator-item/navigator-item-components.tsx

```diff
@@ -250,12 +250,11 @@ const ReplaceElementButton = React.memo((props: ReplaceElementButtonProps) => {
       }}
     >
       <Icn
-        category='tools'
+        category='navigator-element'
```

Comment:
Eventually not all new icons should live in navigator-element (that's for _element_ icons). Not something to fix now, because we don't have a good system in place today either.